helm/adelphi/templates/cassandra-image-builder.yaml

```diff
@@ -44,9 +57,9 @@ spec:
         - sh
         - -c
       args:
-        - wget -O - https://downloads.apache.org/ant/binaries/apache-ant-1.9.15-bin.tar.gz | tar -zxvf -;
-          chmod +x /apache-ant-1.9.15/bin/ant;
-          ln -s /apache-ant-1.9.15/bin/ant /usr/bin/ant;
+        - wget -O - https://downloads.apache.org/ant/binaries/apache-ant-1.10.11-bin.tar.gz | tar -zxvf -;
+          chmod +x /apache-ant-1.10.11/bin/ant;
+          ln -s /apache-ant-1.10.11/bin/ant /usr/bin/ant;
```

Comment:
That Apache downloads page doesn't keep the list of all Ant releases, it seems it only keeps the X most recent ones, so this pod was failing because it couldn't download that version anymore.
We should probably find an suitable image in Dockerhub or publish our own under the adelphitools org.